pkg/genericcomp/genericcomp.go

```diff
@@ -145,3 +160,12 @@ func addContainerEnvs(envs *[]corev1.EnvVar, containers []corev1.Container) {
 		}
 	}
 }
+
+// IsFluentdEnabled checks if Fluentd integration is enabled for this generic component.
+func IsFluentdEnabled(generic *v1beta1v8o.VerrazzanoGenericComponent) bool {
+	if generic.FluentdEnabled == nil {
+		return true
+	}
+
+	return *generic.FluentdEnabled
```

Comment:
Do we document how to disable this?

Reply:
The documentation for generic components is in a future sprint and will need to be documented then.